rai/client_test.go

```diff
@@ -576,58 +566,28 @@ func findOAuthClient(clients []OAuthClient, id string) *OAuthClient {
 		assert.Nil(t, err)
 	}
 
-	rsp, err = client.FindOAuthClient(test.oauthClient)
-	assert.Nil(t, err)
-	assert.Nil(t, rsp)
-
 	rspExtra, err := client.CreateOAuthClient(test.oauthClient, nil)
 	assert.Nil(t, err)
 	assert.Equal(t, test.oauthClient, rspExtra.Name)
 
 	clientID := rspExtra.ID
 
-	rsp, err = client.FindOAuthClient(test.oauthClient)
-	assert.Nil(t, err)
-	assert.NotNil(t, rsp)
-	assert.Equal(t, clientID, rsp.ID)
-	assert.Equal(t, test.oauthClient, rsp.Name)
-
 	rspExtra, err = client.GetOAuthClient(clientID)
 	assert.Nil(t, err)
-	assert.NotNil(t, rsp)
+	assert.NotNil(t, rspExtra)
 	assert.Equal(t, clientID, rspExtra.ID)
 	assert.Equal(t, test.oauthClient, rspExtra.Name)
 
-	clients, err := client.ListOAuthClients()
```

Comment:
it'd be nice to test the list operation somehow (assuming it's eventually consistent), no?

Reply:
Ah I meant to test it in general to make sure there is no regression on listing clients. I agree it's better not to use it for the create oauth client.